### DISCUSSION TITLE
Cant add site

### DISCUSSION THREAD

**Discussion by dandaniel5** [plausible/analytics#4874] [Self-Hosted Support]
I added my site to the cloud version to check if it works.
Now I can’t add it to my CE panel because they say it’s already registered.
Smarty in support says the CE version isn’t related to the cloud version, but that’s nonsense.

I didn’t receive any constructive answers; all they did was blame me for the issue.
However, I have solid proof that the community version checks whether an address has been previously registered in the cloud version. I confirmed this by registering subdomains.

I just need to be able to register my domain in the CE version.
<img width="595" alt="image" src="https://github.com/user-attachments/assets/612b6fbc-73b3-42a6-b34a-ba8f37f05d58">
SASP

**Comment by ruslandoga** [comment]
👋 @dandaniel5 

CE is completely separate from the cloud version :)

You can check which sites have been registered in your CE instance by connecting to PostgreSQL and issuing the following query:

```console
$ cd plausible-ce # or wherever you `git clone`d https://github.com/plausible/community-edition
$ docker compose exec plausible_db psql -d plausible_db -h localhost -U postgres
```
```sql
select * from sites;
```

I also noticed that your instance [allows](https://github.com/plausible/community-edition/wiki/configuration#disable_registration) registrations without invitations, so you might want to check if there are any "stray" users too:

```sql
select * from users;
```

**Reply by ruslandoga** [reply]
> However, I have solid proof that the community version checks whether an address has been previously registered in the cloud version.

The code is open. You can read it and verify that this doesn't happen :)

https://github.com/plausible/analytics/blob/5fec52ab36d249f964785a18e40fc520ad760184/lib/plausible_web/controllers/site_controller.ex#L16-L69

**Reply by dandaniel5** [reply]
I am saying that I cannot add to the CE version something that was previously added to the cloud.

I verified this through practical testing:
First, I checked that I could begin registration for test.com in the CE version. Next, I added test.com to the cloud version by registering it there. After that, I was no longer able to add the same address to the CE version, as it said it was already registered.
Also, I got the same result with wikipedia.com, hbo.com, and live.com. I can register them in CE only if they were not registered in the cloud beforehand.

**Reply by ruslandoga** [reply]
> First, I checked that I could begin registration for test.com in the CE version.

That creates the site. See the code I shared above :)

**Reply by dandaniel5** [reply]
> > First, I checked that I could begin registration for test.com in the CE version.
> 
> That creates the site. See the code I shared above :)



How do you think these 60 lines of code solve my problem? So you think if your code is there, my issue will magically disappear? Or how?

**Reply by ruslandoga** [reply]
Have you tried running the SQL commands I shared? That's what was meant to help you.

The code snippet is just to convince you that CE doesn't check the cloud version for existing sites.

**Reply by dandaniel5** [reply]
> Have you tried running the SQL commands I shared? :)
```
root@sbox:~/community-edition# docker-compose exec plausible_db psql -d plausible_db -h localhost -U postgres
psql (16.5)
Type "help" for help.

plausible_db=# select * from sites;
 id |       domain       |     inserted_at     |     updated_at      |   timezone   | public | locked | has_stats | imported_data | stats_start_date | ingest_rate_limit_scale_seconds | ingest_rate_limit_threshold | native_stats_start_at | domain_changed_from | domain_changed_at | allowed_event_props | conversions_enabled | funnels_enabled | props_enabled | accept_traffic_until |                                                                                                               installation_meta                                                                                                                
----+--------------------+---------------------+---------------------+--------------+--------+--------+-----------+---------------+------------------+---------------------------------+-----------------------------+-----------------------+---------------------+-------------------+---------------------+---------------------+-----------------+---------------+----------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------More-- 
Cancel request sent
plausible_db=# select * from users;
 id |          email           |     inserted_at     |     updated_at      |           name           |      last_seen      |                        password_hash                         | trial_expiry_date | email_verified | theme  | grace_period | previous_email | totp_secret | totp_enabled | totp_last_used_at | allow_next_upgrade_override | totp_token | accept_traffic_until | notes 
----+--------------------------+---------------------+---------------------+--------------------------+---------------------+--------------------------------------------------------------+-------------------+----------------+--------+--------------+----------------+-------------+--------------+-------------------+-----------------------------+------------+----------------------+-------
  7 | da.ndaniels110@gmail.com | 2024-11-21 13:18:36 | 2024-11-21 13:18:36 | Danil                    | 2024-11-21 13:18:36 | $2b$12$EnoHUpF6Txl7V2ZlUe6c8OO5VfI33JewG4Nnpw.GCSbja5Ik1LiAS | 2124-11-21        | t              | system |              |                |             | f            |                   | f                           |      --More-- 
```

**Reply by dandaniel5** [reply]
> Have you tried running the SQL commands I shared? That's what was meant to help you.
> 
> The code snippet is just to convince you that CE doesn't check the cloud version for existing sites.

There’s no way my local PostgreSQL can say that “SITE ALREADY REGISTERED” without checking your cloud database.

**Reply by ruslandoga** [reply]
> There’s no way my local PostgreSQL can say that “SITE ALREADY REGISTERED” without checking your cloud database.

It really doesn't :) But I'm getting a bit tired of this.

The root of the issue is quite obvious from the results you've shared. So you might still be able to resolve it.

Good luck!
